.tours/introducing-lwc-recipes.tour

```diff
@@ -0,0 +1,14 @@
+{
+  "title": "👋 Introducing LWC Recipes",
```

Comment:
probably we want to make sure this states clearly "LWC Recipes OSS"

Reply:
Also, this name is too similar to "Intro to LWC". I know one is intro to the project, and the other is supposed to be intro to the framework. But I'd suggest a different title for one. Like this one could be called "Welcome to LWC OSS Recipes" or something like that. 

Like the :wave: by the way.

Reply:
Done